reference/annotation/acl.rst

```diff
@@ -0,0 +1,95 @@
+@Acl
+====
+
+The ``@Acl`` annotation is used to create a new access control list and to protect the controller
+that is annotation with this ACL accordingly:
+
+.. code-block:: php
+    :linenos:
+
+    /**
+     * @Acl(
+     *     id="user_user_view",
+     *     type="entity",
+     *     class="OroUserBundle:User",
+     *     permission="VIEW"
+     * )
+     */
+    public function demoAction()
+    {
+        // ...
+    }
+
+Options
+-------
+
+``class``
+~~~~~~~~~
+
+**type**: ``string``
+
+When the `type`_ option is set to ``entity``, the ``class`` option is used to decide whether or not
```

Comment:
Please specify what in case if `type` parameter is set to `entity`, this parameter value should be an entity class name